app/src/main/java/org/sopt/dosopttemplate/Extensions.kt

```diff
@@ -67,36 +63,21 @@ val defaultUserInfo = UserInfo(
     "",
     "",
     "",
-    "",
-    LocalDate.of(1000, 5, 11),
-    ""
 )
 
 // Bundle 데이터 받아오기
 
-fun Bundle?.extractUserInfo(): UserInfo {
-    val profileImage = this?.getString("profileImage", "")
-    val userId = this?.getString("userId", "")
-    val password = this?.getString("password", "")
-    val nickName = this?.getString("nickName", "")
-    val MBTI = this?.getString("MBTI", "")
-    val birthdayString = this?.getString("birthday", "")
-    val selfDescription = this?.getString("self_description", "") ?: ""
-
-    val formatter = DateTimeFormatter.ISO_DATE
-    val birthday = if (birthdayString != null && birthdayString.isNotBlank()) {
-        try {
-            LocalDate.parse(birthdayString, formatter)
-        } catch (e: DateTimeParseException) {
-            defaultUserInfo.birthday
-        }
-    } else {
-        defaultUserInfo.birthday
+fun Bundle?.extractUserData(): UserInfoBundle? {
+    if (this == null) {
+        return null
     }
 
-    return if (profileImage != null && userId != null && password != null && nickName != null && MBTI != null) {
-        UserInfo(profileImage, userId, password, nickName, MBTI, birthday, selfDescription)
-    } else {
-        defaultUserInfo
-    }
+    val profileImage = getString("profileImage") ?: ""
```

Comment:
```suggestion
    val profileImage = getString("profileImage").orEmpty()
```
기본 제공되는 확장함수 `orEmpty()`를 사용할 수도 있슴둥

Reply:
아하!